dependencies/pom.xml

```diff
@@ -2,20 +2,20 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <name>Dependency Lister</name>
+  <name>Maven Dependencies</name>
```

Comment:
we should add a description element too. Something like "a library for analyzing Maven repository dependency graphs"

"Maven dependencies" sounds like this is part of Maven, which it isn't. I'm tempted to drop the name element unless we can think of something better. Perhaps even just leave it as Dependency Lister

Reply:
+1

Reply:
Added "A library for analyzing Maven artifact dependency graphs"